WordPress/gutenberg
# [RNMobile] Fix link "open in new tab" switch

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1018

`gutenberg-mobile` side PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1030

This PR fixes the "Open in New Tab" option for links in gutenberg-mobile, where the switch started always off, even a link target was set.

![link-switch](https://user-images.githubusercontent.com/9772967/58329926-0f355d00-7e36-11e9-9dc3-d75a3a6cfd45.gif)

To test:

- Open the example app.
- Select text on a paragraph block and add a link attribute to it.
- Switch the "Open in New Tab" option ON.
- Close the Bottom Sheet.
- Open the Link options again for the same text.
- Check that the Switch is ON.
